### DISCUSSION TITLE
What does this line from urql's pagination docs mean:- They're not meant to implement infinite pagination for any app

### DISCUSSION THREAD

**Discussion by xoldd** [urql-graphql/urql#2600] [Q&A] [answered]
From urql docs :- [link](https://formidable.com/open-source/urql/docs/graphcache/local-resolvers/#pagination)
`They're not meant to implement infinite pagination for any app, instead they're useful when we'd like to add infinite pagination to an app quickly to try it out or if we're unable to replace it with separate components per page in environments like React Native, where a FlatList would require a flat, infinite list of items.`

Does this mean for a real world app I should implement pagination logic by myself instead of using the plugin? Is it not production ready or something?

**Comment by JoviDeCroock** [comment, marked as answer]
Yes, we'll often refer to pagination as a UI-concern. The data is synchronously available through `useQuery` anyway, this is like [our example](https://github.com/FormidableLabs/urql/blob/main/examples/with-pagination/src/PaginatedNpmSearch.jsx#L65).

**Reply by xoldd** [reply]
Thanks got it.
